lib/pbench/server/db/database.py

```diff
@@ -0,0 +1,47 @@
+import sys
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from pbench.server import NoOptionError, NoSectionError
+
+
+class Database:
+    # Create declarative base model that our model can inherit from
```

Comment:
And the "our model" you are referring to is ... ?

Reply:
Our model here refers to the database models we defined with sqlalchemy `UserModel` `MetadataModel` `ActiveTokens` it is defined in my PR https://github.com/distributed-system-analysis/pbench/pull/1937

Reply:
Could we consider:

```
class Database:
    """Database - base class model for use by all other database models.
    """
```

Reply:
This is code I copied from Nikhil's PR originally; I'll have to merge when his goes in, and we can worry about the wording then.